### DISCUSSION TITLE
请问下资源管理中的【Spark/MapReduce用于ETL】如何理解和使用？

### DISCUSSION THREAD

**Discussion by dongchao-1** [apache/doris#31894] [A - General / Q&A]
请问下
https://doris.apache.org/zh-CN/docs/advanced/resource
中的【Spark/MapReduce用于ETL】这种方式如何使用？是将doris的数据转换成spark的rdd可以计算么，还是什么？
有更详细的材料或者demo么？

**Comment by yagagagaga** [comment]
Doris 支持管理两种资源，
一种是 spark 资源，用于 [spark-load](https://doris.apache.org/zh-CN/docs/data-operate/import/import-way/spark-load-manual)，对于doris的数据转换成spark的rdd，可以使用 [spark-connector](https://doris.apache.org/zh-CN/docs/ecosystem/spark-doris-connector)
另一种是 odbc，用于外表（不推荐，已有[catalog](https://doris.apache.org/zh-CN/docs/lakehouse/multi-catalog/jdbc)功能代替）；
